Tabloid/client/src/components/Tag/tagForm.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import { FormGroup, Input, Label, Button, Form } from "reactstrap";
+import { addTag } from "../../modules/tagManager";
+
+const TagForm = () => {
+    const history = useHistory();
+    const emptyTag = {
+        Name: "",
```

Comment:
Watch your casing here. Client side code (especially in React) is camel cased.

Reply:
👍🏻